core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -132,6 +133,12 @@ oppia.directive('explorationSummaryTile', [
               if ($scope.getCollectionId()) {
                 result += ('?collection_id=' + $scope.getCollectionId());
```

Comment:
This should be 
```
    result = UrlService.addParams(result, {
      collection_id: $scope.getCollectionId()
    });
```

Reply:
Done, instead of JSON, I am passing 'collection_id' and in the other case, 'parent', as string.